addons/ai/functions/fnc_garrisonQuantity.sqf

```diff
@@ -118,6 +126,25 @@ for "_i" from 1 to (_aiNumberToSpawn min _freeBuildingSpaces) do {
         _mkr setMarkerText (_unitClassname);
     };
 };
+if((_logic getVariable ["WakeUp", false])) then {
+    {
+        private _building = _x;
+        private _buildingCenter = _building modelToWorld (boundingCenter _building);
+        _buildingCenter set [2,0]; // set to ground
+        private _buildingSize = (boundingBox _building) # 2;
+        private _trigger = createTrigger ["EmptyDetector", _buildingCenter, false];
+        _trigger setVariable ["side", _side];
+        _trigger setVariable ["units", _building getVariable [QGVAR(garrisonedUnits), []]];
+        _trigger setTriggerInterval 2; // set interval to 2 seconds
+        _trigger setTriggerArea [_buildingSize * 3, _buildingSize * 3, 0, false, 10];
+        _trigger setTriggerActivation ["ANYPLAYER","PRESENT",false];
+        _trigger setTriggerStatements [
+            QUOTE(private _side = (thisTrigger getVariable [ARR_2('side', opfor)]); this && ({ side _x != _side} count thisList) > 0),
```

Comment:
Now it isn't needed again

Reply:
Had issues with QQGVAR, so just went back, QUOTE makes the code lintable in my editor so it stays.